spec/README.md

```diff
@@ -79,39 +79,39 @@ A reference to a _term_ looks like this.
 > The provisions of the stability policy are not in effect until
 > the conclusion of the technical preview and adoption of this specification.
 
-Updates to this specification will not change
-the syntactical meaning, the runtime output, or other behaviour
-of valid messages written for earlier versions of this specification
-that only use functions defined in this specification.
+Updates to this specification will not make any well-formed or valid _message_ invalid.
+
 Updates to this specification will not remove any syntax provided in this version.
-Future versions MAY add additional structure or meaning to existing syntax.
+
+Updates to this specification will not change the syntactical meaning
+of any syntax defined in this specification except for that syntax marked as
+"reserved for future standardization".
+
+Updates to this specification will not assign any meaning to or change the syntactical 
+requirements for any private-use annotation.
 
 Updates to this specification will not remove any reserved keywords or sigils.
 
-> [!NOTE]
-> Future versions may define new keywords.
+Updates to this specification will not add any additional Unicode code points to 
+those in `reserved-annotation-start`.
 
-Updates to this specification will not reserve or assign meaning to
-any character "sigils" except for those in the `reserved` production.
+Updates to this specification will not remove any functions defined in the default registry.
 
-Updates to this specification
-will not remove any functions defined in the default registry nor
-will they remove any options or option values.
-Additional options or option values MAY be defined.
+Updates to this specification will not remove any options or option values
+defined in the default registry.
 
 > [!NOTE]
-> This does not guarantee that the results of formatting will never change.
-> Even when the specification doesn't change,
+> The foregoing policies are _not_ a guarantee that the results of formatting will never change.
+> Even when this specification or its implementation do not change,
 > the functions for date formatting, number formatting and so on
```

Comment:
```suggestion
> Even when this specification does not change,
> the implementations of functions for date formatting, number formatting and so on
```

The functions are part of an implementation, so if the behavior of functions changed, it means that one or more implementations changed.

Reply:
Agreed, although I'm also trying to point out that the function implementation (the code) might also remain static and the output change (because your operating system, for example, got a new copy of CLDR data)